scss/example-gut-12.scss

```diff
@@ -0,0 +1,12 @@
+$baseline-impl: (
```

Comment:
Are you sure you need _all_ these properties? Aren't the `$baseline-impl` and `$baseline-default` merged?